console2/src/components/molecules/LogSegment/index.tsx

```diff
@@ -172,7 +172,7 @@ const LogSegment = ({
                     rel="noopener noreferrer"
                     target="_blank"
                     className="AdditionalAction Last"
-                    data-tooltip="Download Log File"
+                    data-tooltip="Download: InstanceId_SegmentId.log"
```

Comment:
Should it be a literal `InstanceId_SegmentId.log` value or it is supposed to be a template?

Reply:
this is just for the users to understand. the actual template is here https://github.com/walmartlabs/concord/pull/223/files#diff-da1f8800427dd0bdac66a3406695f658b0ae05da7621f3eb2e87c180f2b9a3c6R222

Reply:
I mean we could do it like this, but then users will be confused about the filename - `Download: ${instanceId}_${segmentId}.log`

also, printing the actual values causes the tooltip to become really huge.